doc/nrf/app_dev/device_guides/fem/21540ek_dev_guide.rst

```diff
@@ -52,14 +52,20 @@ For example:
 .. parsed-literal::
    :class: highlight
 
-   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dmultiprotocol_rpmsg_SHIELD=nrf21540ek
+   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dipc_radio_SHIELD=nrf21540ek
 
-In this command, the *childImageName_* parameter has the ``multiprotocol_rpmsg_`` value and builds a multiprotocol application with support for 802.15.4 and Bluetooth.
-The *childImageName_* parameter can take the following values:
+In this command, the *childImageName_* parameter has the ``ipc_radio_`` value as default and builds the application with support for the combination of 802.15.4 and Bluetooth.
```

Comment:
AFAIK, child images are going to be removed in 2.9.0 -- any reason we still use this feature here?
https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/config_and_build/multi_image.html

Reply:
I think this `_childImageName` is just a name for second application for the network/radio core here. Id does not refer to the "child image" feature that is deprecated. What better name to pick ?